content/recipes/prisma.md

```diff
@@ -181,6 +181,11 @@ With your Prisma models in place, you can generate your SQL migration files and
 ```bash
 $ npx prisma migrate dev --name init
 ```
+OR for prisma using mongodb
```

Comment:
```suggestion
OR when MongoDB is used:
```

(?)

Reply:
when MongoDB is used, its better... thnks